docs/ManualDeploymentGuideScalarDLAuditorOnAWS.md

```diff
@@ -0,0 +1,174 @@
+
+## Step 1. Create an environment
+
+Scalar DL Auditor component is implemented to detect Byzantine faults,
+it manages the identical states of Ledger so you need to deploy Ledger and Auditor in separate administrative domains. For more details, please refer to [Getting Started with Scalar DL Auditor](https://github.com/scalar-labs/scalardl/blob/master/docs/getting-started-auditor.md) guide.
+
+In this guide, for ease of explanation, we deploy Auditor in a different cluster on the same subscription. However, it is highly recommended to deploy it in another administrative domain in production.
+
+### Requirements
+
+* You must create VPCs with different CIDR ranges.
+
+### Recommendations
+
+* You should deploy Ledger and Auditor in separate administrative domains.
+
+### Steps
+
+* Follow step 1 to step 3 in the [Deploy Scalar DL on AWS](./ManualDeploymentGuideScalarDLOnAWS.md) guide to create a Scalar DL Auditor environment.
+
+## Step 2. Peer the Virtual Networks
+
+This section shows how to peer the VPC for Scalar DL deployment.
+
+In this guide, Ledger and Auditor are deployed on the private subnet of the different VPCs, 
+so you need to add peering for internal communication between the Auditor, Ledger and Client SDK(application).  
+
+### Requirements
+
+* You must create 3 peering between 3 Virtual Networks.
+
+### Recommendations
+* You should create Network ACLs for Ledger and Auditor VPCs.
+* You should restrict all access from the Auditor and Client except scalardl envoy LoadBalancer ports (50051 and 50052) in the Ledger Network ACLs.
+* You should restrict all access from the Ledger and Client except scalardl auditor envoy LoadBalancer ports (40051 and 40052) in the Auditor Network ACLs.
+
+### Steps
+
+* Create the peering between the Ledger and Auditor VPCs on the basis of [AWS official guide](https://docs.aws.amazon.com/vpc/latest/peering/create-vpc-peering-connection.html) with the above requirements.
+* Create the peering between the Ledger and Client VPCs on the basis of [AWS official guide](https://docs.aws.amazon.com/vpc/latest/peering/create-vpc-peering-connection.html) with the above requirements.
+* Create the peering between the Auditor and Client VPCs on the basis of [AWS official guide](https://docs.aws.amazon.com/vpc/latest/peering/create-vpc-peering-connection.html) with the above requirements.
+* Update Ledger route tables with Auditor and Client VPC peering connections based on [AWS official guide](https://docs.aws.amazon.com/vpc/latest/peering/vpc-peering-routing.html).
+* Update Auditor route tables with Ledger and Client VPC peering connections based on [AWS official guide](https://docs.aws.amazon.com/vpc/latest/peering/vpc-peering-routing.html).
+* Update Client route tables with Ledger and Auditor VPC peering connections based on [AWS official guide](https://docs.aws.amazon.com/vpc/latest/peering/vpc-peering-routing.html).
+* Add new Inbound and Outbound rule to restrict unwanted access to Ledger and Auditor.
```

Comment:
These are very unorganized and redundant.

Reply:
I'm sorry, 
 
 Before, Hiro san said `If we need 3 peerings, have 3 steps here. `
  https://github.com/scalar-labs/scalar-kubernetes/pull/150#discussion_r745191191

Reply:
I did not say `have redundant 3 steps here`

Reply:
Thank you for your reply.
Removed redundant sentences.